daisy_workflows/image_import/import_image.sh

```diff
@@ -51,8 +51,12 @@ function resizeDisk() {
   local deviceId="${4}"
 
   echo "Import: Resizing ${diskId} to ${requiredSizeInGb}GB in ${zone}."
-  if ! out=$(gcloud -q compute disks resize ${diskId} --size=${requiredSizeInGb}GB --zone=${zone} 2>&1); then
-    echo "ImportFailed: Failed to resize disk. [Privacy-> resize disk ${diskId} to ${requiredSizeInGb}GB in ${zone}, error: ${out} <-Privacy]"
+  if ! out=$(gcloud -q compute disks resize "${diskId}" --size="${requiredSizeInGb}"GB --zone="${zone}" 2>&1 | tr "\n\r" " "); then
+    if echo "$out" | grep -q "Required .compute.disks.resize. permission"; then
+      echo "ImportFailed: The Compute Engine default service account needs the compute.disks.resize permission."
```

Comment:
Start this message with the failure description like below (failed to resize disk) and then explain what needs to be done to fix it. Saying that compute engine service account needs a permission without an explanation is as confusing as not saying why disk resize failed.

Reply:
sgtm